pkg/must-gather-api.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/konveyor/forklift-must-gather-api/pkg/backend"
 	"gorm.io/gorm"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
 var r *gin.Engine
 var db *gorm.DB
 var corsAllowedOrigins []*regexp.Regexp
 
 func main() {
+	// This line prevents controller-runtime from complaining about log.SetLogger never being called
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
this can be aligned with what we do elsewhere - `logging.Factory.New()`

Reply:
ah I see, logging is an internal package..
ok, so why setting dev-mode to true?